main.js

```diff
@@ -218,8 +218,24 @@ const template = [
           await shell.openExternal('https://rapidpowerestimator.readthedocs.io/en/latest/');
         },
       },
+      {
+        label: 'About',
+        click: () => {
+          // dynamically reading package.json
+          const packagePath = path.join(__dirname, 'package.json');
+          const packageData = JSON.parse(fs.readFileSync(packagePath, 'utf8'));
+      
+          const message = `${packageData.build.productName}\nVersion: ${packageData.version}\nAuthor: ${packageData.author}\nCopyright: ${packageData.build.copyright}\nType: Engineering`;
```

Comment:
Type is still hardcoded here, please remove it.
Also confirm that production build has same data.